pwa/src/templates/documentation/installationDocumentation/InstallationDocumentationTemplate.module.css

```diff
@@ -0,0 +1,22 @@
+:root {
+  --init-size-sm: 12px;
+  --init-size-md: 24px;
+
+  --init-border-radius: 3px;
+
+  --init-color-lightgrey: #f5f5f5;
+}
```

Comment:
As of now, this would be the only place outside of theme and override files where we're adding `:root` rules.

I think we should refactor this to something else, I'd opt for one of the following solutions:

1. Use existing tokens (preferred, if they are available) or
2. Make theme tokens for this.